internal/validation/validation.go

```diff
@@ -60,11 +63,20 @@ func newContext(s *schema.Schema, doc *query.Document, maxDepth int) *context {
 		fieldMap:         make(map[*query.Field]fieldInfo),
 		overlapValidated: make(map[selectionPair]struct{}),
 		maxDepth:         maxDepth,
+		variables:        variables,
 	}
 }
 
 func Validate(s *schema.Schema, doc *query.Document, maxDepth int) []*errors.QueryError {
-	c := newContext(s, doc, maxDepth)
+	return validate(s, doc, maxDepth, bindings{})
```

Comment:
Just pass `nil`, `nil` is a valid empty read-only map, no need to allocate anything here